lib/commands/index.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
-  message: {
+  Message: {
```

Comment:
this should be lowercase as it was

Reply:
I made it uppercase to distinguish with the sign/verify <message> (which is a text that the user sees in the help of the CLI). I'll change that to msg.